README.md

```diff
@@ -16,6 +16,7 @@
 1. Enable DevOps in KubeSphere.
 
    ```shell
+   kubectl delete ns kubesphere-devops-system kubesphere-devops-worker
```

Comment:
I don't think this should be part of the standard steps.

Reply:
Yup, I've deleted the namespace(kubesphere-devops-system and kubesphere-devops-worker) creation process in my ks-installer. but official ks-installer will install those two namespaces automatically, so that we cannot install ks-devops correctly. 

So this is a temporary step here.